### PR TITLE
Optimise app onboarding dialog

### PR DESCRIPTION
**Pull request by nalcalag**
Task/Issue URL: https://app.asana.com/0/0/1203887621832726/f

### Description
Create new dax dialog with a trackers blocked animations which will be used for an experiment to optimise app onboarding

### Steps to test this PR

Asana task with instructions: https://app.asana.com/0/0/1204008611485984/f

_Pixels_
- When new dialog appears -> ☑️ Check pixel `m_odc_s` is fired
- When tapping on dax dialog button -> ☑️ Check pixel `m_odc_ok` is fired
- When tapping on privacy shield modal -> ☑️ Check new pixel `m_onboarding_privacy_shield_button` is fired

### UI changes
| 🌙 Dark Mode  | 🌞 Light Mode |
| ------ | ----- |
https://user-images.githubusercontent.com/20798495/221184911-c2788461-4a15-41df-bf21-8006630d54ef.mp4|https://user-images.githubusercontent.com/20798495/221184929-b198a35a-bd78-4fb0-bb7b-7b2583c5d1d0.mp4

### CODE REVIEW

**Comment by nalcalag** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #2866** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/2866" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/2866?utm_source=stack-comment).

File: statistics/src/main/java/com/duckduckgo/app/statistics/pixels/Pixel.kt [hunk 1]
```diff
@@ -92,6 +92,7 @@ interface Pixel {
         const val DAX_SERP_CTA = "s"
         const val DAX_NETWORK_CTA_1 = "n"
         const val DAX_TRACKERS_BLOCKED_CTA = "t"
+        const val DAX_TRACKERS_BLOCKED_EXPERIMENT_CTA = "te"
```
Comment (marcosholgado) [inline statistics/src/main/java/com/duckduckgo/app/statistics/pixels/Pixel.kt:95]: I don't think you need this. You should be able to use `t` and know which version we are showing by the variant. The issue with having two is that the onboarding never ends (unless we change a few things) and after the experiment is gone, users will start to see `t` again.

Comment (nalcalag) [reply statistics/src/main/java/com/duckduckgo/app/statistics/pixels/Pixel.kt:95]: You are right, I'll remove it and leave `t`

File: app/src/main/java/com/duckduckgo/app/cta/onboarding_experiment/DaxDialogExperiment.kt [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+                primaryCta.text = primaryButtonText
+                dialogText.typingDelayInMs = DEFAULT_TYPING_DELAY
+            }
+        } ?: dismiss()
+    }
+
+    private fun setListeners() {
+        with(binding) {
+            primaryCta.setOnClickListener {
+                dialogText.cancelAnimation()
```
Comment (marcosholgado) [inline app/src/main/java/com/duckduckgo/app/cta/onboarding_experiment/DaxDialogExperiment.kt:150]: We are missing a `daxDialogListener?.onDaxDialogPrimaryCtaClick()` here to also fire the Ok button pixel.

**Review by marcosholgado — APPROVED** [review]
Left a comment as discussed on MM about the Ok button pixel. Apart from that LGTM!

**Pull request merged**